Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "libc-utils"
+version = "0.1.0"
+authors = ["Jing Liu <jing2.liu@linux.intel.com>"]
+edition = "2018"
```

Comment:
Seems a little risky to force 2018 edition.

Reply:
Got it. Remove it and add license BTW.